### PR TITLE
schemas: document schema fixes

### PR DESCRIPTION
**Pull request by kpsherva**

### CODE REVIEW

File: invenio_app_ils/documents/loaders/jsonschemas/document.py [hunk 1]
```diff
@@ -210,7 +210,7 @@ class Meta:
     identifiers = fields.List(fields.Nested(IdentifierSchema))
     imprint = fields.Nested(ImprintSchema)
     internal_notes = fields.List(fields.Nested(InternalNoteSchema))
-    keywords = fields.Nested(KeywordSchema)
+    keywords = fields.List(fields.Nested(KeywordSchema))
```
Comment (ntarocco) [inline invenio_app_ils/documents/loaders/jsonschemas/document.py:213]: Not sure about this change: `keywords` was supposed to be just a text field. Why a list?
We had a long discussion with the library to understand keywords and tags, if you remember.

File: run-style-checks.sh [hunk 2]
```diff
@@ -7,6 +7,6 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 pydocstyle invenio_app_ils tests docs && \
-isort -rc -c -df && \
+isort -rc -c --df && \
```
Comment (ntarocco) [inline run-style-checks.sh]:
```suggestion
isort --check-only --diff && \
```
suggested [here](https://github.com/inveniosoftware/cookiecutter-invenio-instance/pull/238)

File: invenio_app_ils/documents/schemas/documents/document-v1.0.0.json [hunk 3]
```diff
@@ -396,16 +397,20 @@
...
     "keywords": {
-      "properties": {
-        "source": {
-          "type": "string"
-        },
-        "value": {
-          "type": "string"
+      "items": {
+        "additionalProperties": false,
+        "description": "#TODO DISCUSS",
```
Comment (ntarocco) [inline invenio_app_ils/documents/schemas/documents/document-v1.0.0.json]: Discussed? :)

**Review by ntarocco — APPROVED** [review]

File: invenio_app_ils/documents/schemas/documents/document-v1.0.0.json (hunk 3 above)
Comment (kpsherva) [reply invenio_app_ils/documents/schemas/documents/document-v1.0.0.json]: 👀 oh, thanks :D

**Pull request merged**